source/speech.py

```diff
@@ -436,8 +441,16 @@ def getIndentationSpeech(indentation):
 			res.append(symbol)

```

Comment:
It just occurred to me that if we're not going to speak, there's no need to call characterProcessing, etc. We could optimise that out. See if you can do this, but don't bother if the code becomes too unreadable as a result (e.g. too many if checks).